packages/requirements.txt

```diff
@@ -1,2 +1,2 @@
-django==1.2 
+django==1.4.11 
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2014-0482   
**Check Name:** Django: RemoteUserMiddleware session hijacking   
**Severity:** MEDIUM   
**Fixed Version:** 1.4.14, 1.5.9, 1.6.6   
**Description:** The contrib.auth.middleware.RemoteUserMiddleware middleware in Django before 1.4.14, 1.5.x before 1.5.9, 1.6.x before 1.6.6, and 1.7 before release candidate 3, when using the contrib.auth.backends.RemoteUserBackend backend, allows remote authenticated users to hijack web sessions via vectors related to the REMOTE_USER header.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2014-0482